.python-version

```diff
@@ -0,0 +1 @@
+3.10.7
```

Comment:
Can you remove this ?

Reply:
I have removed this. Used `pyenv` in my local so it got included. Should we add it to `.gitignore` instead?